ivy/func_wrapper.py

```diff
@@ -597,17 +598,10 @@ def to_ivy_arrays_and_back(fn: Callable) -> Callable:
     instances and return arrays are all converted to `ivy.NativeArray`
     instances.
     """
-    return output_to_native_arrays(inputs_to_ivy_arrays(fn))
+    return outputs_to_native_arrays(inputs_to_ivy_arrays(fn))
 
 
 def to_native_arrays_and_back(fn: Callable) -> Callable:
```

Comment:
```suggestion
def to_native_arrays_and_back(fn: Callable) -> Callable:
    """
    Make `fn` receive `ivy.NativeArray` and return `ivy.Array`.
    Wrap `fn` so that input arrays are all converted to
    `ivy.NativeArray` instances and return arrays are all converted to
    `ivy.Array` instances.
    """
```